.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,37 @@
+name: BrowserStack Acceptance Tests
+
+on: push
+
+jobs:
+  browserstack-acceptance-tests:
+    name: BrowserStack Acceptance Tests
+    runs-on: ubuntu-latest
+    env:
+      BROWSERSTACK_USERNAME: ${{ secrets.BROWSERSTACK_USERNAME }}
+      BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: npm ci
+      - run: npm run setup-test-site
+      - run: npm run build-test-site
+      - run: chmod +x ./.github/run_browserstack_acceptance.sh
```

Comment:
could we have github track this script as an executable file instead?

Reply:
oh sure, I didn't realize that was possible